components/batch-system/benches/batch-system.rs

```diff
@@ -12,7 +12,7 @@ use std::sync::Arc;
 
 fn end_hook(tx: &std::sync::mpsc::Sender<()>) -> Message {
     let tx = tx.clone();
-    Message::Callback(Box::new(move |_| {
+    Message::Callback(Box::new(move |_, _| {
```

Comment:
Why change it?

Reply:
The signature was changed before and rust-analyzer reports a compilation error.